scripts/scil_tracking_local_dev.py

```diff
@@ -233,8 +233,9 @@ def main():
 
     logging.info("Instantiating propagator.")
     # Converting step size to vox space
-    # We only support iso vox for now.
-    assert odf_sh_res[0] == odf_sh_res[1] == odf_sh_res[2]
+    # We only support iso vox for now but allow slightly different vox 1e-3.
+    assert np.allclose(np.mean(odf_sh_res[:3]),
+                       odf_sh_res[0], atol=1e-03)
```

Comment:
I think this could be change everywhere to:
`np.allclose(np.mean(odf_sh_res[:3]), odf_sh_res, atol=1e-03)`

Why check only the first, when you could check all 3, both will end up being working just fine, so optional comment)